### PR TITLE
Moved multiple dependencies to devDependencies

### LINKED ISSUE #174: Clean dependencies in package.json
Some packages are in dependencies instead of devDependencies such as typescript.

**Task:** going through all dependencies and move unnecessary ones in order to decrease app size 🌱

_NB : due to spam, I won't assigned anyone to this very simple task. First PR up, first to merge, if correct._

### PR DESCRIPTION
**Pull request by maxsynnott**
closes #174

### CODE REVIEW

**Comment by maxsynnott** [comment]
#174

File: package.json [hunk 1]
```diff
@@ -44,12 +44,9 @@
...
     "expo-web-browser": "8.5.0",
-    "front-matter": "3.1.0",
     "glob": "7.1.6",
-    "husky": "4.2.5",
     "i18n-js": "3.5.1",
     "lodash.throttle": "4.1.1",
-    "markdown-it": "10.0.0",
     "moment": "2.24.0",
     "paths-js": "0.4.10",
     "prop-types": "15.7.2",
```
Comment (PierreBresson) [inline package.json:52]: prop-types can be deleted, not use anywhere (legacy)

File: package.json [hunk 2]
```diff
@@ -44,12 +44,9 @@
     "expo-splash-screen": "0.6.1",
     "expo-status-bar": "1.0.2",
     "expo-web-browser": "8.5.0",
-    "front-matter": "3.1.0",
     "glob": "7.1.6",
```
Comment (PierreBresson) [inline package.json:47]: glob should be moved

**Review by PierreBresson — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (maxsynnott) [reply package.json]: Done

File: package.json (hunk 2 above)
Comment (maxsynnott) [reply package.json]: Done

**Review by maxsynnott — COMMENTED** [review]
Changes implemented

**Comment by PierreBresson** [comment]
@maxsynnott Thank you :)

**Pull request merged**